interface-definitions/service_ids_suricata.xml.in

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="ids">
+        <properties>
+          <help>Intrusion Detection System</help>
+        </properties>
+        <children>
+          <node name="suricata" owner="${vyos_conf_scripts_dir}/service_ids_suricata.py">
+            <properties>
+              <help>Suricata high performance Network IDS, IPS and Network Security Monitoring engine</help>
+              <priority>740</priority>
+            </properties>
+            <children>
+              <leafNode name="interface">
+                <properties>
+                  <help>Interfaces to monitor</help>
+                  <completionHelp>
+                    <script>${vyos_completion_dir}/list_interfaces</script>
+                  </completionHelp>
+                  <multi/>
+                </properties>
+              </leafNode>
+              <tagNode name="address-group">
+                <properties>
+                  <help>Address group name</help>
+                  <completionHelp>
+                    <list>HOME_NET EXTERNAL_NET HTTP_SERVERS SMTP_SERVERS SQL_SERVERS DNS_SERVERS TELNET_SERVERS AIM_SERVERS DC_SERVERS DNP3_SERVER DNP3_CLIENT MODBUS_CLIENT MODBUS_SERVER ENIP_CLIENT ENIP_SERVER</list>
```

Comment:
There should be the lowercase list
```
home-net external-net http-server xxx
```
For example, we use it for containers https://github.com/vyos/vyos-1x/blob/f97a3254c9e958f76b3ff3dfec7e9297cf48575c/interface-definitions/container.xml.in#L28

And use `upper` and `replace` if required https://github.com/vyos/vyos-1x/blob/f97a3254c9e958f76b3ff3dfec7e9297cf48575c/src/conf_mode/container.py#L268